kola/tests/docker/docker.go

```diff
@@ -708,3 +710,18 @@ docker run -v "/etc/misc:/opt" --rm ghcr.io/flatcar/busybox true`
 		c.Fatal("/etc/misc/hello should holds 'world'")
 	}
 }
+
+// Reported by a user: a regression on stable
+// that causes ownernship being lost when exporting a Docker
+// container.
+// https://github.com/flatcar/Flatcar/issues/1203
+func dockerOwnership(c cluster.TestCluster) {
```

Comment:
I'm not a large fan of regression tests for mainly upstream issues, specially when they are so specific that they probably won't happen exactly this way again but different. Anyway, if this function won't take long to execute it's ok to have it.